Make sure 'luarocks path' outputs the paths for the Lua ABI version
required by aegisub, not the one used internally by luarocks.

--- a/automation/Makefile
+++ b/automation/Makefile
@@ -19,7 +19,7 @@
 PROGRAM += $(d)aegisub-lua
 
 test-automation: $(PROGRAM)
-	eval `luarocks path`; cd $(TOP)automation; ./aegisub-lua tests/busted.lua -p 'moon' tests/modules
+	eval `luarocks --lua-version 5.1 path`; cd $(TOP)automation; ./aegisub-lua tests/busted.lua -p 'moon' tests/modules
 
 test: test-automation
 
